src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -36,7 +36,7 @@ public SortCommand parse(String args) throws ParseException {
                 PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_STATUS, PREFIX_TAG);
 
         if (prefixes.size() == 0) {
-            throw new ParseException(MESSAGE_NO_PARAMETERS_SUPPLIED);
+            throw new ParseException(String.format(MESSAGE_NO_PARAMETERS_SUPPLIED, SortCommand.MESSAGE_USAGE));
```

Comment:
Acknowledged enhancement for displaying example usages in addition to error message as outlined in #51